modules/vardictjava/main.nf

```diff
@@ -30,7 +30,7 @@ process VARDICTJAVA {
         -th $task.cpus \\
         -N $prefix \\
         -G $reference_fasta \\
-        $regions_of_interest \\
+        $regions_of_interest
```

Comment:
The slashes still need to be there

Reply:
My bad, I'm still getting another error there (that's probably why I didn't notice that yet)